tests/test_controller.py

```diff
@@ -275,3 +275,29 @@ async def ws(self, socket: WebSocket) -> None:
         ws.send_json({"data": "123"})
         data = ws.receive_json()
         assert data
+
+
+@pytest.mark.parametrize(
+    "decorator, http_method, expected_status_code",
+    [
+        (get, HttpMethod.GET, HTTP_200_OK),
+        (post, HttpMethod.POST, HTTP_201_CREATED),
+        (put, HttpMethod.PUT, HTTP_200_OK),
+        (patch, HttpMethod.PATCH, HTTP_200_OK),
+        (delete, HttpMethod.DELETE, HTTP_204_NO_CONTENT),
+    ],
+)
+def test_controller_empty_path(decorator, http_method, expected_status_code):
+    test_path = ""
+
+    class MyController(Controller):
+        path = test_path
+
+        @decorator(path="/")
+        def test_method(self) -> Person:
+            return person_instance
+
+    with pytest.warns(UserWarning), create_test_client(MyController) as client:
+        response = client.request(http_method, "/")
```

Comment:
Seems that it could led to another issue here. if `path=""` then we can't have empty route path.

cc @Goldziher, maybe we can add a second validation here https://github.com/starlite-api/starlite/blob/main/starlite/controller.py#L36-L37 in case `path == ""` and `route_handler.path == ""` 🤷

Reply:
🤦 well handler's path can't be empty https://github.com/starlite-api/starlite/blob/main/starlite/routing.py#L62-L63
so we are fine!